### PR TITLE
Update: Kong Moesif Plugin versions and docs

### PR DESCRIPTION
**Pull request by dgilling**
### Summary
Updates to Kong Moesif plugin docs
Add: Compatible versions
Update: How to install steps
Add: Missing Config options

### Reason
Was out of date

### Testing
Markdown tested 

 review:copyedit

### CODE REVIEW

File: app/_hub/moesif/kong-plugin-moesif/_index.md [hunk 1]
```diff
@@ -141,6 +158,22 @@ params:
       required: false
       default: "`200`"
       description: Maximum batch size when sending to Moesif.
+    - name: user_id_header
+      required: false
+      default: "`X-Consumer-Custom-Id`"
+      description: Request or response header to use to identify the User in Moesif.
```
Comment (Guaris) [inline app/_hub/moesif/kong-plugin-moesif/_index.md:164]:
```suggestion
      description: Request or response header used to identify the User in Moesif.
```

File: app/_hub/moesif/kong-plugin-moesif/_index.md [hunk 2]
```diff
@@ -141,6 +158,22 @@ params:
...
+    - name: company_id_header
+      required: false
+      default: ""
+      description: Request or response header to use to identify the Company (Account) in Moesif.
```
Comment (Guaris) [inline app/_hub/moesif/kong-plugin-moesif/_index.md:168]:
```suggestion
      description: Request or response header used to identify the Company (Account) in Moesif.
```

File: app/_hub/moesif/kong-plugin-moesif/_index.md [hunk 3]
```diff
@@ -189,14 +222,52 @@ This plugin logs to Moesif with an [asynchronous design](https://www.moesif.com/
...
 
-The `.rock` file is a self-contained package that can be installed locally or from a remote server.
+> If you are using Kong's [Kubernetes Ingress Controller](https://github.com/Kong/kubernetes-ingress-controller), the installation is slightly different. Review the [docs for Kubernetes Ingress](https://www.moesif.com/docs/server-integration/kong-ingress-controller/).
+
+The .rock file is a self contained package that can be installed locally or from a remote server.
 
 If the luarocks utility is installed in your system (this is likely the case if you used one of the official installation packages), you can install the 'rock' in your LuaRocks tree (a directory in which LuaRocks installs Lua modules).
 
-It can be installed from luarocks repository by doing:
+### 1. Install the Moesif plugin
```
Comment (Guaris) [inline app/_hub/moesif/kong-plugin-moesif/_index.md:233]:
```suggestion
### Install the Moesif plugin
```

**Review by Guaris — CHANGES_REQUESTED** [review]

**Comment by dgilling** [comment]
Thanks for the feedback @Guaris . Accepted the suggested changes.

**Comment by Guaris** [comment]
I'm closing this in favor of #4007 so I can address some of the changes without pushing directly up to your fork.

File: app/_hub/moesif/kong-plugin-moesif/_index.md [hunk 4]
```diff
@@ -182,35 +215,58 @@ params:
...
 Moesif natively supports REST, GraphQL, Web3, SOAP, JSON-RPC, and more.
 
-### How to install
+## How to install
+
+> If you are using Kong's [Kubernetes Ingress Controller](https://github.com/Kong/kubernetes-ingress-controller), the installation is slightly different. Review the [docs for Kubernetes Ingress](https://www.moesif.com/docs/server-integration/kong-ingress-controller/).
 
 The `.rock` file is a self-contained package that can be installed locally or from a remote server.
 
 If the luarocks utility is installed in your system (this is likely the case if you used one of the official installation packages), you can install the 'rock' in your LuaRocks tree (a directory in which LuaRocks installs Lua modules).
```
Comment (Guaris) [inline app/_hub/moesif/kong-plugin-moesif/_index.md:231]:
```suggestion
If the LuaRocks utility is installed in your system (this is likely the case if you used one of the official installation packages), you can install the 'rock' in your LuaRocks tree (a directory in which LuaRocks installs Lua modules).
```

File: app/_hub/moesif/kong-plugin-moesif/_index.md [hunk 5]
```diff
@@ -182,35 +215,58 @@ params:
...
                                  # loaded via the `bundled` keyword.
 ```
-4. Restart Kong:
+
+If you don't have a `kong.conf`, create one from the default using the following command: 
+`cp /etc/kong/kong.conf.default /etc/kong/kong.conf`
+
+### Restart Kong
+
+After the luarock is installed, restart Kong before enabling the plugin
```
Comment (Guaris) [inline app/_hub/moesif/kong-plugin-moesif/_index.md:254]:
```suggestion
After LuaRocks is installed, restart Kong before enabling the plugin
```

**Review by Guaris — CHANGES_REQUESTED** [review]
I can't push to your fork otherwise I would have done this for you. Could you please also add the word `Moesif` to `dictionary.txt` so that it resembles between `Mockbin` and `MongoDB`?
```
Luarocks
minikube
Mockbin
Moesif
MongoDB
namespace
```

**Comment by dgilling** [comment]
@Guaris , we accepted the changes but were confused on the suggestion around `dictionary.txt`. We don't see any `dictionary.txt` file in this repo. What should we be updating?

**Comment by Guaris** [comment]
> @Guaris , we accepted the changes but were confused on the suggestion around `dictionary.txt`. We don't see any `dictionary.txt` file in this repo. What should we be updating?

The dictionary file is at: 
`.github/styles/kong/dictionary.txt`

Here is a link to the file in your branch: 
https://github.com/Moesif/docs.konghq.com/blob/main/.github/styles/kong/dictionary.txt

**Comment by dgilling** [comment]
@Guaris . Thank you. Just added.

**Comment by Guaris** [comment]
Okay, sorry for the confusion and thank you for your patience, I'm closing this PR in favor of: https://github.com/Kong/docs.konghq.com/pull/4008  which is passing and ready to be merged. This will go in first thing Monday morning.

**Pull request closed without merging**